### PR TITLE
Update name (XCG --> WinUI 2 Gallery)

### PR DESCRIPTION
**Pull request by gregwoo-microsoft**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUI2Gallery/WinUI2Gallery.csproj [hunk 1]
```diff
@@ -1,1320 +1,1320 @@
...
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+    <AppxOSMinVersionReplaceManifestVersion>false</AppxOSMinVersionReplaceManifestVersion>
+    <AppxOSMaxVersionTestedReplaceManifestVersion>false</AppxOSMaxVersionTestedReplaceManifestVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <PackageCertificateKeyFile>XamlControlsGallery_TemporaryKey.pfx</PackageCertificateKeyFile>
```
Comment (chingucoding) [inline WinUI2Gallery/WinUI2Gallery.csproj:21]:
```suggestion
    <PackageCertificateKeyFile>WinUI2Gallery_TemporaryKey.pfx</PackageCertificateKeyFile>
```

**Review by chingucoding — COMMENTED** [review]
Could it be that the new certificate is expired (at least VS says that)

**Comment by gregwoo-microsoft** [comment]
> 

It's possible. Do you suggest that we remove that file entirely?

**Comment by chingucoding** [comment]
I personally would replace it with a new self signed certificate. Adding a self signed cert would make it easier for people downloading the app since they can just download the repo and can hit "run".

**Comment by gregwoo-microsoft** [comment]
Closing this PR as it is a copy

**Pull request closed without merging**